operations/admin.go

```diff
@@ -171,3 +175,50 @@ func setServiceFlagValues(args []string, target bool, flags *model.APIServiceFla
 
 	return catcher.Resolve()
 }
+
+func viewSettings() cli.Command {
+	return cli.Command{
+		Name:   "get-settings",
+		Usage:  "view the evergreen configuration settings",
+		Action: doViewSettings(),
+	}
+}
+
+func doViewSettings() cli.ActionFunc {
+	return func(c *cli.Context) error {
+		confPath := c.Parent().String(confFlagName)
+
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		conf, err := NewClientSetttings(confPath)
+		if err != nil {
+			return errors.Wrap(err, "problem loading configuration")
+		}
+		client := conf.GetRestCommunicator(ctx)
+		defer client.Close()
+
+		settings, err := client.GetSettings(ctx)
+		if err != nil {
+			return errors.Wrap(err, "problem getting evergreen settings")
+		}
+
+		settingsYaml, err := json.MarshalIndent(settings, " ", " ")
+		if err != nil {
+			return errors.Wrap(err, "problem marshalling evergreen settings")
+		}
+
+		sender, err := send.NewPlainLogger("", send.LevelInfo{Default: level.Info, Threshold: level.Debug})
+		if err != nil {
+			return errors.Wrap(err, "problem creating a sender")
+		}
+
+		if err = grip.SetSender(sender); err != nil {
+			return errors.Wrap(err, "problem setting up logger")
+		}
```

Comment:
nit: just use the setPlainLogger before function in the #180 cli.Command struct.